### ISSUE TITLE
contextMenu.item.directive.d.ts(1,31): error TS2307: Cannot find module '@angular/cdk/a11y'

### ISSUE THREAD

**Issue by khanyuriy** [isaacplmann/ngx-contextmenu#71]
Hi, after upgrade, i encountered with this error on compilation.

ERROR in node_modules/ngx-contextmenu/lib/contextMenu.item.directive.d.ts(1,31): error TS2307: Cannot find module '@angular/cdk/a11y'.
node_modules/ngx-contextmenu/lib/contextMenu.service.d.ts(1,60): error TS2307: Cannot find module '@angular/cdk/overlay'.
node_modules/ngx-contextmenu/lib/contextMenuContent.component.d.ts(2,28): error TS2307: Cannot find module '@angular/cdk/overlay'.

I have no dependency on @angular/cdk in my app.

**Comment by isaacplmann** [comment]
What version of `@angular/cdk` did you install?  The latest version of `ngx-contextmenu` relies on `@angular/cdk@2.0.0-beta.12`, but `cdk@5.0.0-rc0` was just released.

**Comment by khanyuriy** [comment]
yep, that's was the case
thanks for your good work!
